### PR TITLE
[Main] Upgrade handler v25 to v26

### PR DESCRIPTION
**Pull request by colmazia**
Fixed: Upgrade handle v2.6

## Implementation details

- implement upgrade handler for v2.5 to v2.6
- add new ica allow msg (group module msg, oracle module msg)
- set globalfee module min gas price to 0.0025

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -919,6 +920,73 @@ func (app *BandApp) setupUpgradeHandlers() {
 			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 		},
 	)
+
+	app.UpgradeKeeper.SetUpgradeHandler(
+		"v2_5",
+		func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			hostParams := icahosttypes.Params{
+				HostEnabled: true,
+				AllowMessages: []string{
```
Comment (RogerKSI) [inline app/app.go:929]: Is it possible to get the current allow messages and add only new one?

File: app/app.go [hunk 2]
```diff
@@ -919,6 +920,73 @@ func (app *BandApp) setupUpgradeHandlers() {
...
+				},
+			}
+			app.ICAHostKeeper.SetParams(ctx, hostParams)
+
+			minGasPriceGenesisState := &globalfeetypes.GenesisState{
+				Params: globalfeetypes.Params{
+					MinimumGasPrices: sdk.DecCoins{sdk.NewDecCoinFromDec("uband", sdk.NewDecWithPrec(25, 4))},
+				},
+			}
+			app.GetSubspace(globalfee.ModuleName).SetParamSet(ctx, &minGasPriceGenesisState.Params)
```
Comment (RogerKSI) [inline app/app.go:982]: should we move this into the module (like initModule from icahost)? so, we can use am.paramSpace.SetParamSet there.

**Comment by RogerKSI** [comment]
please update this to be 2_6

**Comment by colmazia** [comment]
https://hackmd.io/@iMLGovUsSB2tAoup8lKI6g/BJAdGVeg2
https://hackmd.io/@iMLGovUsSB2tAoup8lKI6g/HJlhHPlxh
doc jaaa

File: x/bank/module.go [hunk 3]
```diff
@@ -37,6 +41,15 @@ func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, accountKeeper types.Acc
...
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
+
+	m := bankkeeper.NewMigrator(am.keeper.(keeper.WrappedBankKeeper).Keeper.(bankkeeper.BaseKeeper))
+	if err := cfg.RegisterMigration(types.ModuleName, 1, m.Migrate1to2); err != nil {
+		panic(fmt.Sprintf("failed to migrate x/bank from version 1 to 2: %v", err))
+	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 2, m.Migrate2to3); err != nil {
+		panic(fmt.Sprintf("failed to migrate x/bank from version 2 to 3: %v", err))
+	}
```
Comment (satawatnack) [inline x/bank/module.go:54]: why need to migrate 2 times

Comment (colmazia) [reply x/bank/module.go:54]: I think it would be completed

**Comment by satawatnack** [comment]
> https://hackmd.io/@iMLGovUsSB2tAoup8lKI6g/BJAdGVeg2 https://hackmd.io/@iMLGovUsSB2tAoup8lKI6g/HJlhHPlxh doc jaaa

Check up mainnet upgrade docs didn't change the version

**Review by satawatnack — APPROVED** [review]

**Comment by satawatnack** [comment]
> https://hackmd.io/@iMLGovUsSB2tAoup8lKI6g/BJAdGVeg2 https://hackmd.io/@iMLGovUsSB2tAoup8lKI6g/HJlhHPlxh doc jaaa

the version of 2 docs is not consistent

**Review by taobun — APPROVED** [review]

**Pull request merged**